### ISSUE TITLE
error on yarn run webpack with archive: Failed with exit code 2

### ISSUE THREAD

**Issue by fichterto** [akeneo/pim-community-dev#8965]
I downloaded the archive from http://download.akeneo.com/pim-community-standard-v2.3-latest.tar.gz today.

Installation went well until last command as shown below:

**[akeneo@akeneodev]$** yarn run webpack
yarn run v1.10.1
warning package.json: No license field
$ yarn run sync && NODE_PATH=node_modules webpack --config $npm_package_config_source/webpack.config.js --env=prod
warning package.json: No license field
$ NODE_PATH=node_modules node $npm_package_config_check --source=$npm_package_config_source
Starting webpack from /var/lib/akeneo-pim/pim-community-standard in prod mode

WebpackCleanupPlugin: 0 file(s) deleted.
Version: webpack 3.5.5
Time: 8251ms
              Asset      Size  Chunks                    Chunk Names
         lib.min.js    3.2 MB       0  [emitted]  [big]  lib
      vendor.min.js    883 kB       1  [emitted]  [big]  vendor
        main.min.js  65 bytes       2  [emitted]         main
    manifest.min.js   5.88 kB       3  [emitted]         manifest
     lib.min.js.map   3.83 MB       0  [emitted]         lib
  vendor.min.js.map   1.15 MB       1  [emitted]         vendor
    main.min.js.map  88 bytes       2  [emitted]         main
manifest.min.js.map   5.98 kB       3  [emitted]         manifest

ERROR in ./web/bundles/pimenrich/js/catalog-volume/header.ts
Module parse failed: /var/lib/akeneo-pim/pim-community-standard/web/bundles/pimenrich/js/catalog-volume/header.ts Unexpected token (1:16)
You may need an appropriate loader to handle this file type.
| import BaseView = require('pimenrich/js/view/base');
| import * as _ from 'underscore';
| 
 @ ./web/js/module-registry.js 2:19263-19307
 @ ./vendor/akeneo/pim-community-dev/webpack/require-context.js
 @ ./web/bundles/pimenrich/js/form/builder.js
 @ ./web/bundles/pimenrich/js/index.js
 @ multi babel-polyfill ./web/bundles/pimenrich/js/index.js

ERROR in ./web/bundles/pimenrich/js/catalog-volume/section.ts
Module parse failed: /var/lib/akeneo-pim/pim-community-standard/web/bundles/pimenrich/js/catalog-volume/section.ts Unexpected token (3:16)
You may need an appropriate loader to handle this file type.
| import * as Backbone from 'backbone';
| import * as _ from 'underscore';
| import BaseView = require('pimenrich/js/view/base');
| 
| const __ = require('oro/translator');
 @ ./web/js/module-registry.js 2:19189-19234
 @ ./vendor/akeneo/pim-community-dev/webpack/require-context.js
 @ ./web/bundles/pimenrich/js/form/builder.js
 @ ./web/bundles/pimenrich/js/index.js
 @ multi babel-polyfill ./web/bundles/pimenrich/js/index.js

ERROR in ./web/bundles/pimenrich/js/controller/catalog-volume.ts
Module parse failed: /var/lib/akeneo-pim/pim-community-standard/web/bundles/pimenrich/js/controller/catalog-volume.ts Unexpected token (2:16)
You may need an appropriate loader to handle this file type.
| import * as $ from 'jquery';
| import BaseView = require('pimenrich/js/view/base');
| 
| const BaseController = require('pim/controller/front');
 @ ./web/js/module-registry.js 2:13494-13542
 @ ./vendor/akeneo/pim-community-dev/webpack/require-context.js
 @ ./web/bundles/pimenrich/js/form/builder.js
 @ ./web/bundles/pimenrich/js/index.js
 @ multi babel-polyfill ./web/bundles/pimenrich/js/index.js

ERROR in ./web/bundles/pimenrich/js/i18n.ts
Module parse failed: /var/lib/akeneo-pim/pim-community-standard/web/bundles/pimenrich/js/i18n.ts Unexpected token (1:29)
You may need an appropriate loader to handle this file type.
| const flagTemplate = (country: string, language: string, displayLanguage: boolean) => {
|   return `
| <span class="flag-language">
 @ ./web/bundles/pimenrich/js/formatter/choices/base.js 3:0-74:2
 @ ./web/bundles/pimui/js/pim-initselect2.js
 @ ./web/bundles/oroconfig/js/form/system/tab/localization.js
 @ ./web/js/module-registry.js
 @ ./vendor/akeneo/pim-community-dev/webpack/require-context.js
 @ ./web/bundles/pimenrich/js/form/builder.js
 @ ./web/bundles/pimenrich/js/index.js
 @ multi babel-polyfill ./web/bundles/pimenrich/js/index.js

ERROR in ./web/bundles/pimenrich/js/provider/to-fill-field-provider.ts
Module parse failed: /var/lib/akeneo-pim/pim-community-standard/web/bundles/pimenrich/js/provider/to-fill-field-provider.ts Unexpected token (3:48)
You may need an appropriate loader to handle this file type.
| const _ = require('underscore');
| 
| export const getMissingRequiredFields = (product: any, scope: string, locale: string): string[] => {
|   const scopeMissingAttributes = _.findWhere(product.meta.required_missing_attributes, {channel: scope});
|   if (undefined === scopeMissingAttributes) {
 @ ./web/js/module-registry.js 2:34490-34544
 @ ./vendor/akeneo/pim-community-dev/webpack/require-context.js
 @ ./web/bundles/pimenrich/js/form/builder.js
 @ ./web/bundles/pimenrich/js/index.js
 @ multi babel-polyfill ./web/bundles/pimenrich/js/index.js

ERROR in ./web/bundles/pimenrich/js/view/base.ts
Module parse failed: /var/lib/akeneo-pim/pim-community-standard/web/bundles/pimenrich/js/view/base.ts Unexpected token (15:36)
You may need an appropriate loader to handle this file type.
|  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
|  */
| class BaseView extends Backbone.View<any> implements View {
|   private parent: View | null = null;
|   private extensions: {[code: string]: View};
 @ ./web/bundles/pimdatagrid/js/datagrid/actions-panel.js 10:0-200:2
 @ ./web/js/module-registry.js
 @ ./vendor/akeneo/pim-community-dev/webpack/require-context.js
 @ ./web/bundles/pimenrich/js/form/builder.js
 @ ./web/bundles/pimenrich/js/index.js
 @ multi babel-polyfill ./web/bundles/pimenrich/js/index.js
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by AhmedAlaDali** [comment]
I have the same issues with "yarn run webpack".
Can please anyone from Akeneo help ?

**Comment by Doodoune** [comment]
Hi!

Sorry for the delay, do you still have this issue on the latest archive?

Regards,
Elodie

**Comment by paveq** [comment]
I'm also hitting the same error. So far I'm been unable to conclude what is causing this.

Running Node v11.12.0 and Yarn 1.15.2

**Comment by paveq** [comment]
In our case we hold application code under /usr/src/app directory, and **src** component matches to exclude regex in webpack.config.js file:

`exclude: /lib|node_modules|vendor|tests|src|packages/`

Perhaps there excludes should be relative to the project path?

**Comment by nei** [comment]
I am also getting this error while upgrading to version 3.0.

```
17:56:57 [file+] en_NZ.js
neisantos~/s/akeneo> yarn run webpack                                                                                                                                                   feature/PREZOLA-328-!?
yarn run v1.15.2
warning package.json: No license field
$ yarn run sync && NODE_PATH=node_modules webpack --config vendor/akeneo/pim-community-dev/webpack.config.js --env=prod
warning package.json: No license field
$ NODE_PATH=node_modules node $npm_package_config_check --source=vendor/akeneo/pim-enterprise-dev
Starting webpack from /akeneo in prod mode

WebpackCleanupPlugin: 0 file(s) deleted.
Version: webpack 3.5.5
Time: 123122ms
              Asset       Size  Chunks                    Chunk Names
        main.min.js    1.55 MB       0  [emitted]  [big]  main
         lib.min.js     533 kB       1  [emitted]  [big]  lib
      vendor.min.js  701 bytes       2  [emitted]         vendor
    manifest.min.js    1.45 kB       3  [emitted]         manifest
    main.min.js.map    3.77 MB       0  [emitted]         main
     lib.min.js.map    1.61 MB       1  [emitted]         lib
  vendor.min.js.map    1.76 kB       2  [emitted]         vendor
manifest.min.js.map    7.75 kB       3  [emitted]         manifest

ERROR in ./web/bundles/akeneoasset/js/fields/asset-category-tree.ts
Module parse failed: /akeneo/web/bundles/akeneoasset/js/fields/asset-category-tree.ts Unexpected token (11:5)
You may need an appropriate loader to handle this file type.
|  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
|  */
| type InterfaceNormalizedCategory = {
|   code: string;
|   labels: { [key:string] : string };
 @ ./web/js/module-registry.js 2:93153-93213
 @ ./vendor/akeneo/pim-community-dev/webpack/require-context.js
 @ ./web/bundles/pimui/js/form/builder.js
 @ ./web/bundles/pimui/js/index.js
 @ multi babel-polyfill ./web/bundles/pimui/js/index.js
```

**Comment by nei** [comment]
I've got exacly the same error on a brand new installation of version 3. seems like some dependency missing or some misconfiguration of webpack.

**Comment by Doodoune** [comment]
Hi! 
I see that our product team fix some dependencies recently. 
Do you reproduce this issue with the latest patch?

Regards,

**Comment by nei** [comment]
Can you provide some more details about it @Doodoune  which patch is this? I still trying to fix this issue.

**Comment by Doodoune** [comment]
Hi @nei!
Tamara proposes a solution in the ticket that you raise in our support team (SDS-10569). Can you try to apply the patch that she provide you?

Regards,

**Comment by sugalvojau** [comment]
Same error here.
I have two different environments. The error exists on one of two environments only.

Versions are the same on both environments: 
yarn: 1.5.1
nodejs: v8.11.1
npm: 5.6.0
Using docker image of "node:8.11.1" for both.
Akeneo EE v2.3.35

Suspicions fall on the following on my side, I may be wrong:
- permissions of directories, users, groups, etc.
- environment variables

**Comment by nei** [comment]
@Doodoune Yes I can confirm this PR fixed my issue https://github.com/akeneo/pim-community-dev/pull/9945

**Comment by Doodoune** [comment]
This fix will be available in the next version of the PIM (3.0.15). I will keep you updated.

**Comment by sugalvojau** [comment]
Same here. It works after applying the patch: https://github.com/akeneo/pim-community-dev/pull/9945/files#diff-11e9f7f953edc64ba14b0cc350ae7b9d
Akeneo PIM 2.3.35
Thanks!

**Comment by Doodoune** [comment]
I close this issue as the fix is available in PIM v3.0.15.
Feel free to reopen it if you still have an issue.
Regards,
